tests/index.js

```diff
@@ -1,68 +1,35 @@
-describe('WHEN start component', function() {
-  this.timeout(10000);
-  describe('Interface', function() {
-    it('should have default button label', function(done) {
-      interfaceBrowser
-        .evaluate(function (selector) {
-          return document.querySelector(selector).value;
-        }, 'input[name=name]')
-        .then(function(label) {
-          expect(label).to.equal('Primary button');
-          done();
-        })
+/* global browser */
```

Comment:
I'm imagining every test file is going to need all these global variables declared. Is there an easier way to achieve linting without needing to declare all of them on all the test files in all the components?

Reply:
I moved these declarations to the eslint file